cmd/probe.go

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package cmd
+
+import (
+	"github.com/kubearmor/kubearmor-client/probe"
+	"github.com/kubearmor/kubearmor-client/deployment"
+	"github.com/spf13/cobra"
+	"github.com/fatih/color"
+	"fmt"
+	"bufio"
+	"os"
+)
+
+var probeInstallOptions deployment.ProbeOptions
+
+// probeCmd represents the get command
+var probeCmd = &cobra.Command{
+	Use:   "probe",
+	Short: "Display probe information",
+	Long:  `Display probe information`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		color.Blue("To get full probe, a pod will be deployed in your cluster - This pod will be deleted after probing")
+        color.Blue("Press 1 to get full probing")
+        color.Blue("Press any key to get minimal probing")
+
+		reader := bufio.NewReader(os.Stdin)
+		char, _, err := reader.ReadRune()
+		if err != nil {
+			fmt.Println(err)
+		}
+		if char == '1'{
+			if err := deployment.ProbeDaemonInstaller(client, probeInstallOptions); err != nil {
+				return err
+			}
+
+			if err := probe.PrintFullProbeResult(client); err != nil {
+				return err
+			}
+			if err := deployment.ProbeDaemonUninstaller(client, probeInstallOptions); err != nil {
+				return err
+			}
+			return nil
+		}else{
+			if err := probe.PrintMinimalProbeResult(client); err != nil {
+				return err
+			}
+			return nil
+		}
+
+		
+		
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(probeCmd)
+
+	probeCmd.Flags().StringVarP(&probeInstallOptions.Namespace, "namespace", "n", "default", "Namespace for resources")
+    probeCmd.Flags().StringVarP(&probeInstallOptions.ProbeDaemonImage, "image", "i", "alpine", "Probe daemonset image to use")
```

Comment:
We won't need to change the image here I believe.
But We should add an option to bypass the probe interrupt. Like a `-f|--full` or `-y|--yes` flag which will automatically do a full probing.